<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSettingsWidget</class>
 <widget class="QWidget" name="NetworkSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>625</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="4" column="0">
      <widget class="QLabel" name="lblHost">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblUsername">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="proxyHost"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="proxyUsername"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="proxyPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkUseProxy">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblEnableProxy">
       <property name="text">
        <string>Enable Proxy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string comment="Proxy Type">Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="proxyType">
       <item>
        <property name="text">
         <string>HTTP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SOCKS5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="proxyPort">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkUseProxyForKodi">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblUseProxyForKodi">
       <property name="text">
        <string>Use Proxy for Kodi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkUseProxy</tabstop>
  <tabstop>chkUseProxyForKodi</tabstop>
  <tabstop>proxyType</tabstop>
  <tabstop>proxyHost</tabstop>
  <tabstop>proxyPort</tabstop>
  <tabstop>proxyUsername</tabstop>
  <tabstop>proxyPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
